LoopKit/TemporaryScheduleOverrideHistory.swift

```diff
@@ -68,6 +68,8 @@ public final class TemporaryScheduleOverrideHistory {
     }
     
     private var modificationCounter: Int
+    
+    public var relevantTimeWindow: TimeInterval = TimeInterval.hours(8)
```

Comment:
I see this gets set in Loop, so the default isn't really used, but do you think it's worth changing to 10 hours anyway?